internal/helper/helper.go

```diff
@@ -39,95 +37,18 @@ func InitCommonOpts() (*commands.CommonOpts, *commands.StateMachineOpts) {
 	return new(commands.CommonOpts), new(commands.StateMachineOpts)
 }
 
-// GetHostArch uses dpkg to return the host architecture of the current system
```

Comment:
These have been moved to statemachine/helper.go per the new organization used by sil2100.